api/pkg/apis/v1alpha1/providers/target/staging/staging.go

```diff
@@ -102,9 +104,12 @@ func (i *StagingTargetProvider) Get(ctx context.Context, deployment model.Deploy
 	if scope == "" {
 		scope = "default"
 	}
+	containerName := deployment.Instance.ObjectMeta.Name + "-" + i.Config.TargetName
```

Comment:
We need to use "ResourceSeperator" in api/constants/constants.go instead of hardcoding "-" here. @LeiQL please confirm.

Reply:
This hyphen is not delimiter for versioned resource but the original rule to compose the catalog name.